### PR TITLE
add missing iseq line for eoan

### PR DESCRIPTION
**Pull request by forkiesassds**

### CODE REVIEW

**Comment by antonym** [comment]
Thanks, reason for this is that when the latest version comes out they don’t have the -updates kernel immediately.  We can probably drop the disco line if disco-updates exists now too.

File: src/ubuntu.ipxe [hunk 1]
```diff
@@ -10,7 +10,7 @@ set os Ubuntu
 clear ubuntu_version
 menu ${os} - ${arch_a} - Image Sig Checks: [${img_sigs_enabled}]
 item --gap Latest Releases
-item eoan ${space} ${os} 19.10 Eoan Ermine
+iseq ${arch} x86_64 && item eoan ${space} ${os} 19.10 Eoan Ermine ||
```
Comment (antonym) [inline src/ubuntu.ipxe]: Shouldn’t need this line

File: src/ubuntu.ipxe [hunk 2]
```diff
@@ -33,6 +33,8 @@ goto deb_boot_type
 set mirrorcfg mirror/suite=${ubuntu_version}
 set dir ${ubuntu_base_dir}/dists/${ubuntu_version}-updates/main/installer-${arch_a}/current/images/netboot/
 iseq ${ubuntu_version} disco && set dir ${ubuntu_base_dir}/dists/${ubuntu_version}/main/installer-${arch_a}/current/images/netboot/ ||
```
Comment (antonym) [inline src/ubuntu.ipxe]: Can drop the iseq line for disco

**Review by antonym — CHANGES_REQUESTED** [review]

File: src/ubuntu.ipxe (hunk 1 above)
Comment (forkiesassds) [reply src/ubuntu.ipxe]: 19.10 does not come with a installer-i386 folder. As i386 support was dropped.

File: src/ubuntu.ipxe (hunk 2 above)
Comment (forkiesassds) [reply src/ubuntu.ipxe]: dropped the line for disco

File: src/ubuntu.ipxe (hunk 1 above)
Comment (antonym) [reply src/ubuntu.ipxe]: Ah ok, fair enough, lgtm.

**Review by antonym — APPROVED** [review]

**Pull request merged**